core/services/feeds/service_test.go

```diff
@@ -2189,7 +2190,8 @@ ds1_multiply [type=multiply times=1.23];
 ds1 -> ds1_parse -> ds1_multiply -> answer1;
 answer1      [type=median index=0];
 """
-juelsPerFeeCoinCacheDuration = "30s"
+[pluginConfig.juelsPerFeeCoinCache]
```

Comment:
is this going to break existing specs? if not, why not - it seems like a parsing change

Reply:
We never distributed specs that have this set, default is always used